### PR TITLE
Sign-up component: expire early if the user did not apply.

### PR DESCRIPTION
**Pull request by frankban**

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/6204/
Test PASSed.

**Comment by anthonydillon** [comment]
Code looks good :+1:

File: jujugui/static/gui/src/app/components/deployment-flow/signup/signup.js [hunk 1]
```diff
@@ -59,7 +69,7 @@ YUI.add('deployment-signup', function() {
       @method _handleSignup
     */
     _handleSignup: function() {
-      this._setBetaCookie();
+      this._setBetaCookie(true);
       console.error('Not implemented.');
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/deployment-flow/signup/signup.js]: It is now!

Comment (frankban) [inline jujugui/static/gui/src/app/components/deployment-flow/signup/signup.js]: Not yet. Ant is working on it.

**Comment by frankban** [comment]
Thanks for the reviews!
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**